src/Benchmarks/appsettings.json

```diff
@@ -1,3 +1,5 @@
 ﻿{
-  "ConnectionString": "Server=(localdb)\\mssqllocaldb;Database=aspnet5-Benchmarks;Trusted_Connection=True;MultipleActiveResultSets=true"
+  "ConnectionString": "Server=(localdb)\\mssqllocaldb;Database=aspnet5-Benchmarks;Trusted_Connection=True;MultipleActiveResultSets=true",
+  "EnableAI": true,
+  "AiInstrumenationKey": "6a70388f-9193-463b-9117-4a88ba0e04c6"
```

Comment:
You can remove these, we can pass them by command line args, as it's a source for the app settings. This way we can keep it secret and change it at will.